From 3a2ab4181204386beb7fcecf92b745d420fe1ecb Mon Sep 17 00:00:00 2001
From: Hugo <hugovk@users.noreply.github.com>
Date: Tue, 24 Jul 2018 13:03:48 +0300
Subject: [PATCH] Update supported Python versions (#34)

* Drop support for EOL Python 2.6

* Add Trove classifiers and python_requires
---
 .travis.yml |  1 -
 setup.py    | 11 +++++++++++
 tox.ini     |  2 +-
 3 files changed, 12 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 98488e9..e0eaca0 100644
--- a/setup.py
+++ b/setup.py
@@ -24,4 +24,15 @@
     long_description=open('README.rst').read(),
     packages=find_packages(exclude=['tests.*', 'tests']),
     include_package_data=True,
+    classifiers=[
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: Implementation :: CPython',
+    ],
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
 )
